packages/browser-sdk/package.json

```diff
@@ -33,6 +33,9 @@
     "typescript": "^5.0.4"
   },
   "dependencies": {
+    "@solana/compat": "2.1.1",
+    "@solana/kit": "^2.1.1",
+    "@solana/transactions": "^2.1.1",
     "@solana/web3.js": "^1.98.2"
```

Comment:
We discussed offline, would be nice to not bring in web3.js as a dependency or at least make sure if importing types only keeps web3.js from getting included in bundles.